scripts/vulnerabilities.sh

```diff
@@ -91,17 +78,12 @@ echo -n "Scanning" >&2
 # Sum the fixable vulnerabilities with severity CRITICAL, HIGH, or MEDIUM
 for image in "${images[@]}"; do
   echo -n "."
-  if [[ "${image}" == "gcr.io/${GCP_PROJECT}/"* ]]; then
```

Comment:
This check was filtering out images that aren't re-hosted in our project. They have to be in our project for image scanning to work.

Reply:
They don't need to be in a single project for this to work. The assumptions here are that:
- all images should be hosted either in GCR/GAR
- all image registry should have container scanning enabled

I've tested this locally with the current postsubmit artifacts and it works as expected

Reply:
Does `gcloud beta container images describe` work for both GCR and GAR? I thought they had different gcloud commands for the vuln scanner features.

Reply:
Seems to work for both GCR and GAR from my testing. AFAICT they use the same container analysis api